### DISCUSSION TITLE
ABC Australia iView problems

### DISCUSSION THREAD

**Discussion by makilahy** [aclap-dev/video-downloadhelper#443] [Help and Q&A]
### Your question

I'm trying to download a video from this selection at ABC Australia: https://iview.abc.net.au/show/catalyst/series/0/video/SC2002H010S00

However, the videos I download come out as gibberish. Video is garbled, and there is no audio. I've even tried connecting through a VPN pointed to Australia, but I get the same result.

There's another thread about the abc.net.au site (https://github.com/aclap-dev/video-downloadhelper/discussions/265), but after reading through it, I don't think it's the same problem.

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: false, browser: edge
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36 Edg/124.0.0.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\makil",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

Your description of what you downloaded sounds like it's protected by DRM.  When something is protected by DRM, there's an icon that appears at the top of the browser window to the left of the URL that indicates there's DRM content on the page.  Hover your mouse over the various icons up there & you'll probably find it.  Do report back whether that's what you discover.  VDH has never, does not now, & will never download content protected by DRM.  The VDH folks have posted multiple times in this forum & in our old Google forum that they have no intention of breaking DRM protection.

Used to be that screen recorders (like OBS) could at least record content protected by DRM.  But I recently went through a rather lengthy process of evaluating screen recorders & discovered that there has been a change in the landscape.  The screen recording crowd has colluded with the content providers.  Most screen recorders now recognize DRM protection & refuse to record the content.  All you get is a video with a black screen.  There are services, meaning you have to pay for them, that will record content even if it is protected by DRM.  The few I stumbled across were not cheap.  Most of them offer subscriptions, monthly or annual, not licenses, meaning they want you to pay & pay & pay.

So this isn't a bug in VDH.  It's your web site being a jerk.  Forget about trying to download the content, unless you can cost justify one of those DRM-breaking services.  I think you can guess that I can't cost justify it.

**Reply by GrampaWildWilly** [reply]
#265 was something completely different, as John Cleese would say.  That was an audio-only site, & after Paul fixed something, VDH worked just fine there.  Actually, VDH did work just fine there even before Paul's fix but it wasn't easy to figure out how to do it.  In any case, VDH's success over there proves there was no issue with DRM there.  It sounds like you've got a DRM issue.

**Reply by makilahy** [reply]
I didn't see any icon as you described, but DRM makes sense. I suspected that might be the case.

No, it's not terribly important. I just like to archive good educational material for my personal reference, since good things don't stay long on the internet. It's definitely not worth paying for a hack.
